protocol-designer/src/components/LiquidsPage/LiquidEditForm.tsx

```diff
@@ -84,81 +99,97 @@ export function LiquidEditForm(props: Props): JSX.Element {
-        <Card className={styles.form_card}>
-          <form onSubmit={handleSubmit}>
-            <section className={styles.section}>
-              <div className={formStyles.header}>
-                {i18n.t('form.liquid_edit.details')}
-              </div>
-              <div className={formStyles.row_container}>
-                <FormGroup
-                  label={i18n.t('form.liquid_edit.name')}
-                  className={formStyles.column}
-                >
-                  <InputField
-                    name="name"
-                    error={touched.name ? errors.name : null}
-                    value={values.name}
-                    onChange={handleChange}
-                    onBlur={handleBlur}
-                  />
-                </FormGroup>
-                <FormGroup
-                  label={i18n.t('form.liquid_edit.description')}
-                  className={formStyles.column}
+      }: FormikProps<LiquidEditFormValues>) => {
+        return (
+          <Card className={styles.form_card}>
+            <form onSubmit={handleSubmit}>
+              <section className={styles.section}>
+                <div className={formStyles.header}>
+                  {i18n.t('form.liquid_edit.details')}
+                </div>
+                <div className={formStyles.row_container}>
+                  <FormGroup
+                    label={i18n.t('form.liquid_edit.name')}
+                    className={formStyles.column}
+                  >
+                    <InputField
+                      name="name"
+                      error={touched.name ? errors.name : null}
+                      value={values.name}
+                      onChange={handleChange}
+                      onBlur={handleBlur}
+                    />
+                  </FormGroup>
+                  <FormGroup
+                    label={i18n.t('form.liquid_edit.description')}
+                    className={formStyles.column}
+                  >
+                    <InputField
+                      name="description"
+                      value={values.description}
+                      onChange={handleChange}
+                    />
+                  </FormGroup>
+                  <FormGroup label={i18n.t('form.liquid_edit.displayColor')}>
+                    <Field
+                      name="displayColor"
+                      component={ColorPicker}
+                      value={values.displayColor}
+                      onChange={(color: ColorResult['hex']) => {
+                        setFieldValue('displayColor', color)
+                      }}
+                    />
+                  </FormGroup>
+                </div>
+                <Flex
+                  justifyContent={JUSTIFY_END}
+                  color="#9e5e00"
+                  fontSize="0.625rem"
+                  fontWeight={TYPOGRAPHY.fontWeightSemiBold}
+                  marginTop={errors.name != null ? `-0.25rem` : `0rem`}
```

Comment:
```suggestion
                  marginTop={errors.name != null ? '-0.25rem' : '0rem'}
```

If we could avoid using `-`, that would be great. But I guess this is really needed and the easiest way.